DataConnectors/Agari/AzureFunctionAgari/run.ps1

```diff
@@ -182,20 +182,22 @@ if ($bpEnabled){
         $bpHeaders = @{
                     'Authorization' = "Bearer $bpToken"
                     'ContentType' = 'application/json'
+                    'x-api-requestby' = 'Sentinel/Powershell'
```

Comment:
Is this replacing the user-agent?
Is there a reason we cannot include a user-agent in the Agari API requests?

Reply:
Yes, they require it for the auto-deployment

Reply:
Just re-tested. We can use 'UserAgent' = 'Sentinel/Powershell' or something like that - I was using the wrong string before

Reply:
Would prefer to use a three-part string if possible. That is what we use in our other integrations.
The parts are:
app name
integration description / version
hosting application / version

E.g.
AgariSentinel  BP_APD_Integration/v1.0  SentinelPowerShell/v2.3

Ideally the Sentinel version could be acquired programatically.